SUSE/scf
# check_swapaccounting: fix condition

## Description

We were testing for string non-emptiness (which was always true) rather than string contents, which means that we were not correctly disabling diego-cell when swap is available but swap accounting was not on.

Fixes #2684

## Test plan

- Set up vagrant with swap on, but swapaccounting off
- Try to deploy, it should fail
- Turn off the swap, and try again
- It should pass.
